icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -254,6 +254,60 @@ void dumpEnumeratedProperty(UProperty uproperty, FILE* f) {
     usrc_writeUCPTrie(f, shortPropName, utrie.getAlias(), UPRV_TARGET_SYNTAX_TOML);
 }
 
+/*
+* Export Bidi_Mirrored_Glyph values (code points) in a similar way to how enumerated
+* properties are dumped to file.
+* Note: the data will store 0 for code points without a value defined for
+* Bidi_Mirrored_Glyph.
+*/
+void dumpBidiMirroredGlyph(UProperty uproperty, FILE* f) {
```

Comment:
Is it useful to have this function, which is specific for one single property, take a UProperty parameter?
The other per-single-property functions don't.